build.gradle

```diff
@@ -10,7 +10,7 @@ buildscript {
         robolectricVersion = '4.3.1'
         jUnitVersion = '4.12'
         jSoupVersion = '1.11.3'
-        wordpressUtilsVersion = '1.21'
+        wordpressUtilsVersion = '1.30'
```

Comment:
Hi! I just checked the version of the current WordPress Utils and it is [`1.30.1`](https://github.com/wordpress-mobile/WordPress-Utils-Android/releases/tag/1.30.1) so this version would need to be updated so that the build can pass. What do you think?

Reply:
Done